internal/cli/context_inspect.go

```diff
@@ -0,0 +1,98 @@
+package cli
+
+import (
+	"encoding/json"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	"github.com/posener/complete"
+)
+
+type ContextInspectCommand struct {
+	*baseCommand
+
+	flagJson bool
+}
+
+func (c *ContextInspectCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	flagSet := c.Flags()
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(flagSet),
+		WithNoConfig(),
+		WithClient(false),
+	); err != nil {
+		return 1
+	}
+
+	def, err := c.contextStorage.Default()
+	if err != nil {
+		def = "<unknown>"
+	}
+
+	if c.flagJson {
+		data, err := json.MarshalIndent(map[string]interface{}{
+			"config_path":     c.homeConfigPath,
+			"default_context": def,
+		}, "", "  ")
+		if err != nil {
+			c.ui.Output("Error rendering json: %s", err)
+			return 1
+		}
+
+		c.ui.Output(string(data))
+		return 0
+	}
+
+	c.ui.Output("Context Info:", terminal.WithHeaderStyle())
+
+	if def == "" {
+		def = "<unset>"
+	}
+
+	c.ui.NamedValues([]terminal.NamedValue{
+		{
+			Name: "config path", Value: c.homeConfigPath,
+		},
+		{
+			Name: "default context", Value: def,
+		},
+	}, terminal.WithInfoStyle())
+
+	return 0
+}
+
+func (c *ContextInspectCommand) Flags() *flag.Sets {
+	return c.flagSet(0, func(set *flag.Sets) {
+		f := set.NewSet("Command Options")
+
+		f.BoolVar(&flag.BoolVar{
+			Name:    "json",
+			Target:  &c.flagJson,
+			Usage:   "Output information in JSON format",
+			Default: false,
+		})
+	})
+}
+
+func (c *ContextInspectCommand) AutocompleteArgs() complete.Predictor {
+	return complete.PredictNothing
+}
+
+func (c *ContextInspectCommand) AutocompleteFlags() complete.Flags {
+	return c.Flags().Completions()
+}
+
+func (c *ContextInspectCommand) Synopsis() string {
+	return "Output current context info."
+}
+
+func (c *ContextInspectCommand) Help() string {
+	return formatHelp(`
+Usage: waypoint context inspect <name>=<value>
```

Comment:
What is `name` and `value` here? Maybe this was accidentally left in since I don't think this CLI command takes an argument?

Reply:
It could be nice if if accepted a context name though so you wouldn't have to switch the current context to see the context info:

```
Usage: waypoint context inspect <context_name>
```

Reply:
Actually maybe it's `[context_name]` I don't remember the accepted way to convey optional arguments for CLI doc help strings 😅

Reply:
This might only be worthwhile if we end up showing more information with `inspect` now that I've written all of that out loud....

Since we're only showing default context and config path I don't think that makes sense, so I think just fixing up the doc string is good 👍🏻

Reply:
Oh bad copypasta, just need to remove it.

Reply:
I added the ability to inspect a specific context too.